packages/blockchain/src/consensus/casper.ts

```diff
@@ -14,5 +15,11 @@ export class CasperConsensus implements Consensus {
   public async genesisInit(): Promise<void> {}
   public async setup(): Promise<void> {}
   public async validate(): Promise<void> {}
+  public async validateDifficulty(header: BlockHeader): Promise<void> {
```

Comment:
Can we have some new lines between all the methods here (actually also the empty ones)? This looks pretty crambled.